### PR TITLE
better benchmarks

### PR DESCRIPTION
**Pull request by Uzlopak**
I think this is more expressive

native: empty  x 188,286,109 ops/sec ±0.19% (193 runs sampled)
fast-array-join: empty  x 238,564,443 ops/sec ±0.11% (195 runs sampled)
native: single x 40,731,565 ops/sec ±2.20% (188 runs sampled)
fast-array-join: single x 242,383,990 ops/sec ±0.08% (194 runs sampled)
native: small  x 16,294,118 ops/sec ±1.59% (192 runs sampled)
fast-array-join: small  x 51,757,109 ops/sec ±0.18% (193 runs sampled)
native: large  x 89,448 ops/sec ±0.55% (188 runs sampled)
fast-array-join: large  x 161,832 ops/sec ±2.79% (191 runs sampled)

### CODE REVIEW

File: benchmark/bench-thread.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  case "native":
+    suite.add(`${benchmark.type}: ${benchmark.name}`, () => {
+      benchmark.input.join();
+    });
+    break;
+}
+
+suite.on("cycle", (event) => {
+  parentPort.postMessage(String(event.target));
+}).run();
```
Comment (anonrig) [inline benchmark/bench-thread.js]: Can you add the missing end of line chars?

File: benchmark/benchmark.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  for (const benchmark of benchmarks) {
+    benchmark.name = benchmark.name.padEnd(maxNameLength, " ");
+    const nativeResultMessage = await runBenchmark({ type: 'native', ...benchmark });
+    console.log(nativeResultMessage);
+    const resultMessage = await runBenchmark({ type: 'fast-array-join', ...benchmark });
+    console.log(resultMessage);
+  }
+}
+
+runBenchmarks();
```
Comment (anonrig) [inline benchmark/benchmark.js]: Can you add the missing end of line chars?

**Review by anonrig — CHANGES_REQUESTED** [review]

File: benchmark/benchmark.js (hunk 2 above)
Comment (Uzlopak) [reply benchmark/benchmark.js]:
```suggestion
runBenchmarks();

```

File: benchmark/bench-thread.js (hunk 1 above)
Comment (Uzlopak) [reply benchmark/bench-thread.js]:
```suggestion
}).run();

```

**Pull request merged**